presto-main/src/main/java/io/prestosql/sql/planner/optimizations/HashGenerationOptimizer.java

```diff
@@ -98,11 +97,6 @@
 {
     public static final int INITIAL_HASH_VALUE = 0;
     private static final String HASH_CODE = mangleOperatorName(OperatorType.HASH_CODE);
-    private static final Signature COMBINE_HASH = new Signature(
```

Comment:
Please remove this in a separate commit (in this PR).

Reply:
Added that change in this commit 97852d6